types/react-native/index.d.ts

```diff
@@ -9407,6 +9407,19 @@ export interface KeyboardStatic extends NativeEventEmitter {
      * position changes with keyboard movements.
      */
     scheduleLayoutAnimation: (event: KeyboardEvent) => void;
+    /**
+     * Removes a specific listener.
+     *
+     * @param {string} eventName The `nativeEvent` is the string that identifies the event you're listening for.
+     * @param {function} callback function to be called when the event fires.
+     */
+    removeListener: (eventName: KeyboardEventName, callback: KeyboardEventListener) => void;
```

Comment:
I can find [`removeAllListeners`](https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/Libraries/Components/Keyboard/Keyboard.js#L159) in the React Native source code, but I couldn’t find `removeListener`. Is this potentially not supported anymore as [this comment indicates](https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/Libraries/Components/Keyboard/Keyboard.js#L144) or am I looking at the wrong place?

Reply:
It's a little complicated. React Native v0.64, which is current stable version, has `removeListener` but the deprecation of this function is planned in the next version.